core/azure-core/src/main/java/com/azure/core/polling/PollerOptions.java

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.polling;
+
+/**
+ * Place to provide configuration options for Poller
+ **/
+public class PollerOptions {
+
+    private String negativeValueFormat = "Negative or zero not allowed.Provided value %d";
+    private String timeoutSmallerThanPollIntervalFormat = "Poll Interval %d should be smaller than timeout %d";
+    private String growthFactorValueFormat = "Growth factor %.5f should be equal or larger than 1.0";
+
+    private int timeoutInMilliSeconds;
```

Comment:
The timeout requirement has just been removed from the spec: https://github.com/Azure/azure-sdk/commit/685b6901dad84258818dc26790311044145a2a73